src/style.css

```diff
@@ -0,0 +1,111 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+}
+
+header {
+  width: 100%;
+  background-color: #664c06;
+  gap: 5rem;
+  justify-content: center;
+  margin-bottom: 2rem;
+  font-weight: 800;
+  display: flex;
+  font-size: 16px;
+  border-bottom: #333 solid 1px;
+}
+
+.menu-header {
+  text-align: center;
+  justify-content: center;
+}
+
+#grid-container {
+  justify-content: center;
+  width: 30%;
+}
+
+.img-fluid {
+  width: 300px;
+}
+
+.column {
+  float: left;
+  width: 25%;
+  padding: 0 10px;
+}
+
+.feedback {
+  float: right;
+}
+
+.food-list {
+  margin-left: 5%;
+  justify-content: center;
+}
+
+.food-image {
+  width: 100%;
+}
+
+/* .row:after {
+  content: "";
+  display: table;
+  clear: both;
+} */
```

Comment:
- [ ] Please remove commented code from your code base as it make you code base dirty and less readable. Please kindly fix that.